boards/arc/emsdp/platform.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2023 Synopsys
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/arch/cpu.h>
+#include <zephyr/init.h>
+
+#define DFSS_SPI0_BASE			0x80010000
+#define DFSS_SPI1_BASE			0x80010100
+#define DW_SPI_REG_CLK_ENA		(0x16)  /* DFSS only */
+
+static int emsdp_dfss_clock_init(void)
+{
+	sys_out32(1, DFSS_SPI0_BASE + DW_SPI_REG_CLK_ENA);
```

Comment:
Even though from used defines one may conclude what's being done here, I'd suggest to add comments like "Enable clock for DFSS SPI0 controller" & "Enable clock for DFSS SPI1 controller".